open-craft/ansible-playbooks
# BB-1464: Updates chkrootkit expected output file after unattended upgrades

This PR adds a apt hook to update the chkrootkit expected output file after a system upgrade.
This is required to avoid useless spam email on the ops mailing list.

**Concerns:**
* I'm not sure, but this could leave an unprotected window if unattended-upgrades runs before chkrootkit.

**Testing instructions:**
1. Checkout this branch, and make changes to `common-server` and `common-server-init` to only deploy the `chkrootkit.yml` task.
2. Run the playbook.
3. SSH into the VM and install a software using apt (e.g.: `apt install sl`).
4. Check that there is a message like this in apt's output: 
```
Updating chkrootkit expected output...
```

You can also check **haproxy-stage-1**, I tested this role there.

**Reviewer:**
